cmd/ci-secret-bootstrap/main.go

```diff
@@ -419,10 +419,6 @@ func updateSecrets(secretsGetters map[string]coreclientset.SecretsGetter, secret
 			logrus.Debugf("handling secret: %s:%s/%s", cluster, secret.Namespace, secret.Name)
 			secretsGetter := secretsGetters[cluster]
 			if existingSecret, err := secretsGetter.Secrets(secret.Namespace).Get(context.TODO(), secret.Name, meta.GetOptions{}); err == nil {
-				if secret.Type != existingSecret.Type {
```

Comment:
This field is immutable, that is why the check exists. You need to recognize this situation and recreate the secret, rather than updating it

Reply:
Yes I checked that too. I will update this PR to recreate the secret instead. Patching will make sense here.